Index: django_evolution/db/__init__.py
===================================================================
--- django_evolution/db/__init__.py	(revision 183)
+++ django_evolution/db/__init__.py	(working copy)
@@ -2,7 +2,11 @@
 
 from django.conf import settings
 
-module_name = ['django_evolution.db',settings.DATABASE_ENGINE]
+if settings.DATABASE_ENGINE: # django 1.1 support
+    module_name = ['django_evolution.db',settings.DATABASE_ENGINE]
+else: # django 1.2 support
+    module_name = ['django_evolution.db', settings.DATABASES['default']['ENGINE'].split('.')[3]]
+
 module = __import__('.'.join(module_name),{},{},[''])
 
 evolver = module.EvolutionOperations()
